packages/@aws-cdk/core/README.md

```diff
@@ -92,6 +92,36 @@ nested stack and referenced using `Fn::GetAtt "Outputs.Xxx"` from the parent.
 
 Nested stacks also support the use of Docker image and file assets.
 
+### Breaking automatic references
```

Comment:
Feels like this should be part of the section that describes [cross references](https://docs.aws.amazon.com/cdk/latest/guide/resources.html#resource_stack) in the dev guide.

Maybe we can copy the contents from the dev guide to here and just add this as a subsection? And then also duplicate this to the dev guide?